R/net.mod.init.R

```diff
@@ -176,22 +188,23 @@ init_status.net <- function(dat) {
         }
       }
     }
-    dat$attr$status <- status
+    dat <- set_attr(dat, "status", status)
   } else {
-    status <- dat$attr$status
+    #TODO: Where is status set before this? NULL if statonNW = TRUE
```

Comment:
Fix, remove?

Reply:
Updated. However: looking through the code it looks like if initial conditions are passed through init.net and status isn't in the formation model (but is in on the network) then status (on the network) is ignored? Is this correct? Should the status on the network only be pulled (eg: through copy_nwattr_to_datarr) if status is in the formation model AND no initial conditions are passed?

Reply:
I think status passed on the network, whether in the formation model or not, should override  any initial conditions passed through init.net. ok?

Reply:
statOnNw check has now been updated to look for status on vertex attr. and not whether in the model or not.